app/cli/cmd/attestation_add.go

```diff
@@ -90,13 +100,12 @@ func newAttestationAddCmd() *cobra.Command {
 	}
 
 	cmd.Flags().StringVar(&name, "name", "", "name of the material to be recorded")
-	err := cmd.MarkFlagRequired("name")
-	cobra.CheckErr(err)
 	cmd.Flags().StringVar(&value, "value", "", "value to be recorded")
-	err = cmd.MarkFlagRequired("value")
+	err := cmd.MarkFlagRequired("value")
 	cobra.CheckErr(err)
 	cmd.Flags().StringSliceVar(&annotationsFlag, "annotation", nil, "additional annotation in the format of key=value")
 	flagAttestationID(cmd)
+	cmd.Flags().StringVar(&kind, "kind", "", "kind of the material to be recorded")
```

Comment:
could we show here what options are available?

I believe you did something similar already somewhere else.

Reply:
Yep, I'll add them

Reply:
+1 I think we can print the list of supported material types. It should be easy from the proto enum.